tools/setup

```diff
@@ -64,6 +64,8 @@ if [ "$OS_NAME" = "linux" ]; then
 elif [ "$OS_NAME" = "osx" ]; then
   status "0.1. Checking for Command Line Tools"
   xcode-select --version || xcode-select --install || fail "Unable to install Command Line Tools"
+  status "0.2. Updating SSL Certificates for Python3.6"
+  /Applications/Python\ 3.6/Install\ Certificates.command
```

Comment:
Is it always Python\ 3.6?  Maybe there's a way to find /Applications/Python\ 3.* and run it? If not, this'll be good enuf, until our macbooks get force upgraded.

Reply:
I'm happy to kick the can down the road for now :)

Reply:
https://xkcd.com/1205/